qrack/benchmarks.cc

```diff
@@ -0,0 +1,81 @@
+#include <benchmark/benchmark.h>
+#include "qrack/qfactory.hpp"
+
+#include <algorithm>
+#include <iostream>
+
+using namespace Qrack; 
+
+auto min_estimator = [](const std::vector<double>& v) -> double {return *(std::min_element(std::begin(v), std::end(v)));};
+
+static void BM_sim_X(benchmark::State& state) {
+	for (auto _ : state){
+		QInterfacePtr qReg = CreateQuantumInterface(QINTERFACE_QUNIT, QINTERFACE_OPTIMAL, state.range(0), 0);
```

Comment:
allocate the register outside the benchmark loop, or it will measure the allocations time as well.